<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCodScreenShot</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	font-family : Tahoma;
    min-height: 20px;
}
QWidget#widget{
	background-color: rgb(250, 250, 250);
	border : 1px solid gray;
}
#b_exit, #b_screenshot, #b_setting, #b_timer {
    border: none;
}
#b_exit:hover, #b_screenshot:hover, #b_setting:hover, #b_timer:hover {
    background-color: #eeeeee;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="QPushButton" name="startBtn">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>120</y>
         <width>75</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>开始录制</string>
       </property>
      </widget>
      <widget class="QPushButton" name="stopBtn">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>120</y>
         <width>75</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>停止录制</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>80</y>
         <width>54</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>保存路径:</string>
       </property>
      </widget>
      <widget class="QLabel" name="pathLabel">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>80</y>
         <width>311</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>path</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pathBtn">
       <property name="geometry">
        <rect>
         <x>51</x>
         <y>121</y>
         <width>75</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>更改路径</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>0</y>
         <width>401</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>22</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Drag to move window</string>
          </property>
          <property name="styleSheet">
           <string notr="true">:hover {
	image: url(:/new/icon/resource/handle.png);
    background-color: #eeeeee;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>resource/handle.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="b_screenshot">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">image: url(:/new/icon/resource/camera.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>resource/camera.png</normaloff>resource/camera.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="b_timer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">
image: url(:/new/icon/resource/timer.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>resource/timer.png</normaloff>resource/timer.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="b_setting">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">image: url(:/new/icon/resource/settings.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>resource/settings.png</normaloff>resource/settings.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="b_exit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">image: url(:/new/icon/resource/door_out.png);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>resource/door_out.png</normaloff>resource/door_out.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
